monidlistener/monidlistener.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+import os
+import re
+import urllib.parse
+
+from redbot.core import checks, commands, data_manager, Config
+
+import discord
+import tsutils
+
+
+class MonIdListener(commands.Cog):
+    """Monster Name from ID"""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.config = Config.get_conf(self, identifier=10100779)
```

Comment:
You need to initialize values for channels with `self.config.register_channel(enabled=False)` which adds `enabled` as a value for each channel and sets the default value for every channel as "False"